src/cal.py

```diff
@@ -21,4 +21,23 @@
 
 import sys
 import calendar
-from datetime import datetime
\ No newline at end of file
+from datetime import datetime
+# import datetime
+# text = calendar.TextCalendar() 
+
+# def createCal():
+#   x = calendar.TextCalendar()
+#   print(x.formatmonth(2016, 2))
+
+def createCal():
+  try: 
+    month = input("Enter month: ")
+    year = input("Enter year: ")
+    cl = calendar.TextCalendar()
```

Comment:
these are actually great Nick, your use of using try/except and in my opinion is a lot more efficient than the traditional if statements, in this sort of scenario, keep up the good work!